src/header.jsx

```diff
@@ -115,3 +119,47 @@ const ViewSelector = ({ isGrid, setIsGrid, sortBy, setSortBy }) => {
         </Select>
     );
 };
+
+const UploadButton = ({ currentDir, files }) => {
+    const ref = useRef();
+
+    const handleClick = () => {
+        ref.current.click();
+    };
+
+    const onUpload = e => {
+        const uploadedFile = e.target.files[0];
+
+        let fileName = uploadedFile.name;
+
+        // eslint-disable-next-line no-loop-func
+        while (files.some(f => f.name === fileName)) {
+            fileName += " (1)";
+        }
```

Comment:
This would create files like "name (1) (1) (1)". Shouldn't this rather count up,  like "name (3)"?